algviz/graphs/graphs.py

```diff
@@ -0,0 +1,36 @@
+import pygraphviz as pgv
+
+
+def nodesToLocations(nodes, edges, algo="dot"):
```

Comment:
You should extend this so that a user can pass a dictionary of attributes for nodes/edges to `nodesToLocations` that will then be passed as kwargs to the `G.add_node` and `G.add_edge` calls.